test/jasmine/tests/plots_test.js

```diff
@@ -287,6 +287,17 @@ describe('Test Plots', function() {
                 expect(svgHeight).toBe(400);
             }
         });
+
+        it('should update the axis scales', function() {
+            var fullLayout = gd._fullLayout,
+                plotinfo = fullLayout._plots.xy;
+
+            expect(fullLayout.xaxis._length).toEqual(240);
+            expect(fullLayout.yaxis._length).toEqual(220);
+
+            expect(plotinfo.xaxis._length).toEqual(240);
+            expect(plotinfo.yaxis._length).toEqual(220);
```

Comment:
In particular @rreusser , these assertions were failing before https://github.com/plotly/plotly.js/pull/1049/commits/f4ca110ac0b786eed371fa680cdd95c09d5225a2